erc/ERCS/erc-7562.md

```diff
@@ -317,19 +317,24 @@ These are storage rules that help to protect the bundler from denial of service:
   This means that `Paymaster` and `Factory` contracts cannot practically be an "account" contract as well.
 * **[STO-041]** `UserOperation` may not use associated storage (of either its account or from staked entity) in a contract that is a "sender" of another UserOperation in the mempool.
 
-### Staked Entities Reputation Rules
 
-* **[SREP-010]** The "canonical mempool" defines a staked entity if it has `MIN_STAKE_VALUE` and unstake delay of `MIN_UNSTAKE_DELAY`
-* **[SREP-020]** A `BANNED` address is not allowed into the mempool.\
+### General Reputation rules
+The following reputation rules apply for both staked and unstaked entities.
```

Comment:
Define which entities this applies to:
- All staked entities
- Unstaked paymasters (but not other unstaked entities)

Reply:
Fixed